any_member_test.cpp

```diff
@@ -103,7 +103,7 @@ std::ostream& operator<<(std::ostream& os, X const& x) {return os << x.str() <<
 class S
     :public MemberSymbolTable<S>
 {
-    friend class any_member_test;
+    friend struct any_member_test;
```

Comment:
Please remove this commit from this PR, it's unrelated and I shouldn't have included it in the original one. TIA!